packages/ui5-application-writer/src/data/index.ts

```diff
@@ -24,6 +24,11 @@ export function mergeWithDefaults(ui5App: Ui5App): {
     }
     ui5App.ui5 = mergeUi5(ui5App.ui5 || {}, ui5App.appOptions);
     ui5App.package = mergeObjects(packageDefaults(ui5App.package.version, ui5App.app.description), ui5App.package);
+    // if typescript is enabled and pre scripts for start and build scripts
```

Comment:
Independent of my comment about not adding them at, if we add them, shouldn't they be added here: https://github.com/SAP/open-ux-tools/blob/tbi/754/type_check_scripts/packages/ui5-application-writer/templates/optional/typescript/package.json#L3

Reply:
Oh yeah, definitely. Thanks. Done in 6ee2ff2994b682f6eb18d4e9137844cce291c257